src/Microsoft.AspNetCore.Routing/Tree/TreeRouter.cs

```diff
@@ -179,7 +177,7 @@ public VirtualPathData GetVirtualPath(VirtualPathContext context)
                 var tokenizer = new PathTokenizer(context.HttpContext.Request.Path);
                 var root = tree.Root;
 
-                var treeEnumerator = new TreeEnumerator(root, tokenizer);
+                var treeEnumerator = new Treenumerator(root, tokenizer);
```

Comment:
`E` shortage?

Reply:
I think it's better this way, but we can change if I'm only one who holds that opinion.

Reply:
I appreciate what you're doing but I'd prefer the proper name :grimacing: